docs/introduction/Cross-chain-transfer-protocol.rst

```diff
@@ -209,6 +209,26 @@ Mainchain request can be performed through a raw transaction with the following
             "scProof": sc_proof1
         }]
 
+Circuit with key rotation
+===========================
+
+Circuit with key rotation is needed to replace compromised signers and masters keys of certificate submitters with new keys.
+These changes occur in-chain for 2 reasons:
+- every node must keep knowledge about the recent set of public keys. And if we keep this information off-chain we can easily loose it.
+- we need to be sure that all the nodes use exactly the same source of data for signing or verifying certificate, creating the snark proof, etc.
```

Comment:
Lets put indentation at the beginning of every line(at least 4 spaces).

Reply:
Done